gluten-arrow/src/main/scala/org/apache/gluten/runtime/Runtimes.scala

```diff
@@ -29,4 +31,17 @@ object Runtimes {
       s"$backendName:$name",
       () => Runtime(backendName, name))
   }
+
+  def newInstance(
```

Comment:
If `addResourceIfNotRegistered` is called then the mothod name should be `contextInstance` for accuracy. Then we can make the other method `contextInstance(backendName: String, name: String)` a shortcut to this one, or merge the two methods.